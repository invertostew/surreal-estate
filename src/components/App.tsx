import React from "react";

import Header from "./Header";

import "../styles/App.css";

function App() {
  return (
    <main className="surreal-estate-app">
      <Header />
    </main>
  );
}

export default App;
