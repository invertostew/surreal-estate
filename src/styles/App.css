.surreal-estate {
  display: grid;
  grid-template-columns: 15vw 1fr;
  min-height: calc(100vh - 112px);
}
