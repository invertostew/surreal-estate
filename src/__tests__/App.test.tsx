import React from "react";
import { render, screen } from "@testing-library/react";

import App from "../components/App";

test("renders the h2", () => {
  render(<App />);
  const h2El = screen.getByText(/surreal estate/i);
  expect(h2El).toBeInTheDocument();
});
