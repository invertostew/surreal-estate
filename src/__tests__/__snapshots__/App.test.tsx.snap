// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <main
    class="surreal-estate-app"
  >
    <header
      class="header"
    >
      <div
        class="header-logo"
      >
        <img
          alt="Surreal Estate"
          class="header-logo__logo"
          src="#"
        />
      </div>
      <nav
        class="navbar"
      >
        <ul
          class="navbar-links"
        >
          <li
            class="navbar-links__item"
          >
            <a
              href="#!"
            >
              View Properties
            </a>
          </li>
          <li
            class="navbar-links__item"
          >
            <a
              href="#!"
            >
              Add a Property
            </a>
          </li>
        </ul>
      </nav>
    </header>
  </main>
</DocumentFragment>
`;
